databay/inlets/http_inlet.py

```diff
@@ -49,9 +68,10 @@ async def pull(self, update) -> Union[List[Record], str]:
         :return: Single or multiple records produced.
         :rtype: :any:`Record` or list[:any:`Record`]
         """
+        if self.context is not None: self.tcp_connector = aiohttp.TCPConnector(ssl=self.context)
         _LOGGER.info(f'{update} pulling {self.url}')
-        async with aiohttp.ClientSession() as session:
-            async with session.get(self.url) as response:
+        async with aiohttp.ClientSession(connector=self.tcp_connector) as session:
```

Comment:
All of this looks good, I'm assuming it's okay to set `connector=None` in cases where self.tcp_connector is none. It should just silently pass over it.

Reply:
Good check man, yes - the default parameter is `None` so it's safe to pass it as such 👍